src/app/landing/CovidInfoModal.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import Modal from '../../components/modal';
+import Header from '../../components/header';
+import Button from '../../components/button';
+import Icon from '../../components/icon';
+
+function CovidInfoModal({ history, closeModal }) {
+  const toMap = (e) => {
+    e.preventDefault();
+    history.push('/find');
+  };
+
+  return (
+    <Modal compact>
+      <div className="covidTopBar">
+        <div className="m-4"><img src="/img/virus.png" alt="virus icon" /></div>
+        <div onClick={closeModal} className="closeBtn">
+          <Icon name="times" />
```

Comment:
The up-to-date designs don't include a way to close the modal, and I believe that's because once the user has clicked "Get Started", the thinking is that they're already "in transition" to the map.
In fact, it would've probably made sense for the modal to be on top of the map and not the landing page (so the locations can load in the background while the user is reading it), but whatever.

Anyway, I don't think there's a reason to let the user go back, as if they've already clicked the CTA, we want them to get into the actual app. And of course, once they're there, they can always just navigate back.

If you feel differently, though, that could be discussed with Di.